### PR TITLE
fix doc typo and format

### PR DESCRIPTION
**Pull request by silverling**
### PR types

Bug fixes
### PR changes

Docs
### Description

涉及到的 API 有
- paddle.zeros
- paddle.nn.ParameterList
- paddle.nn.BatchNorm
- paddle.nn.BatchNorm1D

这是「[飞桨快乐开源活动](https://github.com/PaddlePaddle/Paddle/issues/56689)」的热身任务一的 PR.

### CODE REVIEW

**Review by sunzhongkai588 — APPROVED** [review]
LGTM for docs
